CWSpices/CWMain.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+require __DIR__ . '/Spices/Spice.php';
+require __DIR__ . '/Spices/SpiceCollection.php';
+require __DIR__ . '/Spices/Spice/Salt.php';
```

Comment:
Ja reiz ir pievienots cikls, kas salādē iekšā kodu, tad šis jau veidos dublikātus, un es apšaubu, ka kods strādā bez kļūdām. Salt un Vegeta klases tiek 2x ielādētas.
Minimālais bija 3.
CTRL+ALT+L lai noformatētu kodu pirms iesniegšanas!